sdcm/cluster.py

```diff
@@ -3836,6 +3846,7 @@ def configure_alert_manager(self, node):
     def restart_scylla_monitoring(self, sct_metrics=False):
         for node in self.nodes:
             self.stop_scylla_monitoring(node)
+            self.stop_selenium_remote_webdriver(node)
```

Comment:
why do we need to stop start it ? with monitoring ?

Reply:
Don't need. Missed it. thanks.